src/components/Footer.jsx

```diff
@@ -0,0 +1,58 @@
+import { Link } from "react-router-dom";
+import "../styles/footer.css"
+
+export function Footer() {
+  return (
+    <div class="py-md footer footer-doc">
```

Comment:
In React we use  `className` instead of `class`

Reply:
Yes I have not refactored my code from HTML to JSX, have done so now, thanks!